dev/org.eclipse.codewind.ui/src/org/eclipse/codewind/ui/internal/messages/messages.properties

```diff
@@ -150,14 +150,15 @@ MoveProjectTimeout=A timeout occurred while moving the {0} project from {1} to {
 
 ProjectDeployedDialogShell=Manage Deployments
 ProjectDeployedDialogTitle=Project Already Deployed
-ProjectDeployedDialogMessage=The project at location {0} is already deployed and enabled on other connections. It is not recommended to have the same project on multiple connections.
-ProjectDeployedDialogGroupLabel=Choose the behaviour for existing enabled deployments
-ProjectDeployedDialogRemoveLabel=Remove
-ProjectDeployedDialogRemoveTooltip=Remove any existing enabled deployments
-ProjectDeployedDialogDisableLabel=Disable
-ProjectDeployedDialogDisableTooltip=Disable any existing enabled deployments
-ProjectDeployedDialogMaintainLabel=Maintain
-ProjectDeployedDialogMaintainTooltip=Do not make any changes to existing enabled deployments
+ProjectDeployedDialogMessage=The project at location {0} is already deployed and enabled on other connections.
+ProjectDeployedDialogInstructions=The project is already deployed on connections: {0}. It is not recommended to have the same project deployed on multiple connections. Choose whether to remove, disable or maintain the existing deployments of this project. Click Cancel to go back and select a different project.
```

Comment:
What about..."The project is already deployed on the following connections: {0}. Remove, disable, or maintain the existing deployments of this project. Click Cancel to go back and select a different project."

- Would it be OK to remove the "It is not recommended..." sentence? Or could we mention why users should avoid deploying the same project on multiple connections? Something like, "A project might fail if it's deployed on multiple connections."

Reply:
Thanks for the review. I have updated it according to your suggestion.